pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -136,8 +135,8 @@ private static long getRandomInitialDelayInSeconds() {
   private static final String REALTIME_SEGMENT_METADATA_COMMIT_NUMLOCKS =
       "controller.realtime.segment.metadata.commit.numLocks";
   private static final String ENABLE_STORAGE_QUOTA_CHECK = "controller.enable.storage.quota.check";
-
   private static final String ENABLE_BATCH_MESSAGE_MODE = "controller.enable.batch.message.mode";
+  private static final String ALLOW_HLC_TABLES = "controller.allow.hlc.tables";
```

Comment:
please add a paragraph to describe what happns when this is turned off.

Reply:
Added.